src/bin/pgcopydb/dump_restore.c

```diff
@@ -86,6 +87,48 @@ copydb_objectid_has_been_processed_already(CopyDataSpec *specs,
 }
 
 
+/**
+ * Stores the section options in the provided 'sections' array based on
+ * the given PostgresDumpSection. In the end, the 'sections' will be a
+ * NULL terminated array.
+ *
+ * @param section The PostgresDumpSection value representing the section.
+ * @param sections The array to store the section options.
+ */
+void
+getSectionOptions(PostgresDumpSection section, char *sections[3])
+{
+	switch (section)
+	{
+		case PG_DUMP_SECTION_SCHEMA | PG_DUMP_SECTION_ALL:
+		{
+			sections[0] = "pre-data";
+			sections[1] = "post-data";
+			break;
+		}
```

Comment:
I do not think that's needed. The pg_dump documented way to do that is to use `--schema-only`, I believe.